examples/react-native-sdk-demo-app/package.json

```diff
@@ -28,6 +28,7 @@
     "react-native-get-random-values": "~1.11.0",
     "react-native-safe-area-context": "5.4.0",
     "react-native-screens": "~4.11.1",
```

Comment:
This package supplies the native navigation primitives that might conflict with react native modal